openlcsd/flow/tasks.py

```diff
@@ -221,6 +221,28 @@ def download_package_archive(context, engine):
     engine.logger.info('Finished downloading source.')
 
 
+def download_component(context, engine):
+    """
+    Download the component src from brew. If the component's src was
+    downloaded from source container, then no need to download source of the
+    package archive from brew.
+    """
+    config = context.get('config')
+    nvr = context.get('package_nvr')
+    src_dir = context.get('src_dir')
+    if not context.get('src_dir'):
+        download_package_archive(context, engine)
+    else:
+        sc_handler = SourceContainerHandler(config)
+        if os.path.isdir(src_dir):
+            if 'rpm_dir' in src_dir:
```

Comment:
could you elaborate what does `rpm_dir` in `src_dir` mean? how would `rpm_dir` makes any difference considering this is inside `download_component`?

Reply:
In the unpack source container stage, the srpm files are located in the directory of rpm_dir. 
If the import task params has the src_dir, that means it's no need to download the source. If the rpm_dir is in the src_dir, that means the source is srpm of the container.